src/components/sidebar/helpers/generate-resources-nav-items.ts

```diff
@@ -0,0 +1,40 @@
+import { ProductSlug } from 'types/products'
+
+const DEFAULT_COMMUNITY_FORUM_LINK = 'https://discuss.hashicorp.com/'
+
+const COMMUNITY_FORUM_LINKS_BY_SLUG = {
+  vault: 'https://discuss.hashicorp.com/c/vault/30',
+  waypoint: 'https://discuss.hashicorp.com/c/waypoint/51',
+}
```

Comment:
Can we easily access `currentProduct` here? This seems like something that would be well suited to living in our product data files!

Reply:
Unfortunately no, since `useCurrentProduct` has to be used in a React component. I considered putting this in the data files where they were before, but opted against it because we probably don't want these to be easily authorable. Before, we allowed all custom links for this section, which meant we enabled inconsistent content for this section by product. I think this approach gives us the control we need to provide a consistent experience of this section of the sidebar.